web-app/css/imosTheme.css

```diff
@@ -337,5 +337,14 @@ ul.x-tab-strip-top {
     border-radius: 6px;
 }
 
+.subsetPanelAccordionItem .x-tool:not(:first-of-type) {
+    margin: 3px 0;
+}
+
+.subsetPanelAccordionItem .x-tool:first-of-type {
+    margin-left: 6px;
+    margin-right: 0;
+}
+
```

Comment:
Okay, cool. I didn't look into how these changes affect things, but if they're not required for the thing to work then they're fine in here as you say.